README.md

```diff
@@ -118,10 +89,14 @@ psql -d foia -c "CREATE USER foia WITH PASSWORD '<<PASSWORD>>';"
 django-admin.py syncdb
 ```
 
-* Finally, launch the server locally:
+* Update the `DATABASE_URL` in `.env` with your Postgres connection string.
+
+### Running the server
```

Comment:
I think the 'running the server' section should come after the 'loading the data' section.